Amēlija.x

```diff
@@ -110,12 +110,11 @@ static void loadPrefs() {
 
 
 
-%hook AXNView
+static NSInteger (*origNumberOfCells)(id self, SEL _cmd, id collectionView, NSInteger section);
 
+NSInteger numberOfCells(id self, SEL _cmd, id collectionView, NSInteger section){
```

Comment:
Hello fellow contributor! Thank you for the PR! 🎉 What a coincidence, this was just what I needed! Anyways, before merging.. I'll ask you to please separate that appended brace, also found in line 127. Thank you!!!

Reply:
@Exerra hmm what do you think of this behavior? bad right? He refused to come here and fix this atrocity because he was too lazy to sign in to GH from mobile. Now I'll have to do it myself because I don't want to keep my users waiting on the new update due to this :uhh:

Reply:
I, the individual known as Exerra, am bound by a contractual obligation to the individual known as @Luki120 to send a message agreeing with the individual known as @Luki120, so I shall send it here.

@RuntimeOverflow you dummy do what mr stalin says

Reply:
Lmao you make me hate you and have a good laugh at the same time aaa